docs/OncoTree-Mapping-Tool.md

```diff
@@ -30,163 +30,163 @@ The list of OncoTree versions available are viewable [here](http://oncotree.mskc
 
-Source OncoTree code `ALL` maps directly to codes `BLL` and `TLL`. Users should place either `BLL` or `TLL` in the `ONCOTRE_CODE`column. The `ONCOTREE_CODE_OPTIONS` column also notes that more granular choices were introduced; as such, users can use the summary file for additional guidance. 
+Source OncoTree code `ALL` maps directly to codes `BLL` and `TLL`. Users should place either `BLL` or `TLL` in the `ONCOTRE_CODE`column. The `ONCOTREE_CODE_OPTIONS` column also notes that more granular choices were introduced; as such, users can use the summary file for additional guidance.
 
 Searching by source code, the following information can be found in the summary file:
-![ALL-example](https://raw.githubusercontent.com/averyniceday/oncotree/doc-expansion/docs/images/ALL-example.png)  
+![ALL-example](images/ALL-example.png)
 The summary file provides a link to the closest shared parent node `LNM`; users can choose more granular nodes by referencing the provided tree:
-![LNM-tree](https://raw.githubusercontent.com/averyniceday/oncotree/doc-expansion/docs/images/LNM-tree.png)  
+![LNM-tree](images/LNM-tree.png)
 
-###### Sample 2  
+###### Sample 2
 
 |SAMPLE_ID|ONCOTREE_CODE  |ONCOTREE_CODE_OPTIONS|
 |:--------|:-------------:|:----------------------:|
 |S1       |	              |BALL -> {BLL}, more granular choices introduced|
-  
+
 Source OncoTree code `BALL` maps directly to `BLL`. Users should place `BLL` in the `ONCOTREE_CODE` column. However, similar to sample 1, the `ONCOTREE_CODE_OPTIONS` indicates there are more granular choices available. Users can follow the same steps as above and use the summary file to select a more granular node.
- 
-###### Sample 4  
+
+###### Sample 4
 
 |SAMPLE_ID|ONCOTREE_CODE  |ONCOTREE_CODE_OPTIONS|
 |:--------|:-------------:|:----------------------:|
-|S4       |	PTCL          |                        |  
-  
-No additional resolution is needed; the previous OncoTree code was already automatically mapped to PTCL and placed in the `ONCOTREE_CODE` column. `ONCOTREE_CODE_OPTIONS` is empty because no manual selections were necessary.  
+|S4       |	PTCL          |                        |
 
-###### Sample 9 
+No additional resolution is needed; the previous OncoTree code was already automatically mapped to PTCL and placed in the `ONCOTREE_CODE` column. `ONCOTREE_CODE_OPTIONS` is empty because no manual selections were necessary.
+
+###### Sample 9
 
 |SAMPLE_ID|ONCOTREE_CODE  |ONCOTREE_CODE_OPTIONS|
 |:--------|:-------------:|:----------------------:|
-|S4       |	              |HGNEE -> ???, Oncotee code not in source oncotree version|  
-  
-Source OncoTree code `HGNEE` was not found in the source OncoTree version and therefore could not be mapped. Users can either reassign a new source OncoTree code (and rerun the script) or remove the sample.  
+|S4       |	              |HGNEE -> ???, Oncotee code not in source oncotree version|
```

Comment:
Typo - "Oncotee" -> "Oncotree"

Reply:
fixed here, and in code (this was incorrect in the mapping script too)